### ISSUE TITLE
[Bug]: Music can't be turned off

### ISSUE THREAD

**Issue by EnderKill98** [BVengo/sound-controller#10]
### Describe the Bug

It seems that this mod causes the vanilla slider for Music to be ignored. I usually turn it completely off and was confused when it suddenly started playing. Putting the slider to 0 did nothing.

### Steps to Reproduce

Go into `Settings -> Music & Sound Options` and set `Music` to `OFF` (I did it from the main menu, not the in-game options).

### Expected Behavior

No background music should be playing.

### Screenshots

![image](https://github.com/BVengo/sound-controller/assets/117233381/00def43a-2444-4db9-b950-5cda25b6f2f1)

### Fabric Version

0.15.10

### Mod Version

1.1.0

### Log Output

Not relly relevant, but this is what I found in the logs, relating to the mod:

```shell
 - Mod 'Sound Controller' (soundcontroller) 1.1.0 recommends version 10.0.0 or later of mod 'Mod Menu' (modmenu), but only the wrong version is present: 10.0.0-beta.1!
[18:13:54] [Render thread/INFO] (Minecraft) Sound engine started
```

### Additional Context

Props for porting so fast to 1.20.5, though! :+1:

**Comment by BVengo** [comment]
Thanks for opening an issue!

I'll try to get this fixed as soon as I'm free, because it's definitely an important bug.

In the meantime, are you able to confirm if the other global sliders work as expected? (No worries if you don't get the chance either - it would only save me a couple of seconds later).

**Comment by EnderKill98** [comment]
I did some ingame checks (singleplayer, as I don't have a permanent test server for 1.20.5 ready, yet).

Music seems to work ingame. Just not outside (title screen, options menu from title screen, etc.).

I did some testing with chickens and the Friendly Mobs slider. It seems to work, but I'm noticing a slight delay in volume adjustment:
 - Turning Friendly mob sounds off
 - Spawning a bunch of friendly mobs
 - I can sometimes hear like about one tick of the sound, before it gets cut off/muted
